soc/arc/snps_nsim/Kconfig

```diff
@@ -15,6 +15,11 @@ config SOC_NSIM_EM7D_V22
 	select CPU_HAS_MPU
 	select CPU_HAS_FPU
 
+config SOC_NSIM_EM11D
+	bool "Synopsys ARC EM_DSP in nSIM"
+	select CPU_HAS_MPU
+	select CPU_HAS_DSP
```

Comment:
no HAS_AGU_REGS here? (I see they are later configured in the em11d specific config - why DSP here, and AGU there?).

Reply:
My first thinking is DSP is a kind of feature just like FPU or MPU. But AGU is more like registers, just like `CONFIG_ARC_HAS_ACCL_REGS`.

Reply:
That's ok - the choice is a bit arbitrary. AGU is for me more a feature as well instead of 'more registers', more like a DMA engine (autonomously loads/stores data from registers with auto-incrementing addresses).